modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc

```diff
@@ -259,7 +261,7 @@ output application/json
 payload
 ----
 +
-* Save and run the project as a Mule Application.
+. Save and run the project as a Mule Application.
```

Comment:
GLOBAL. No one else init. caps. this.
If you are representing a syntax value or UI label, then....
```suggestion
. Save and run the project as a Mule application.
```